.prettierrc.js

```diff
@@ -5,6 +5,5 @@ module.exports = {
   printWidth: 80,
   tabWidth: 2,
   arrowParens: 'avoid',
-  svelteSortOrder: 'options-scripts-styles-markup',
-  plugins: ['prettier-plugin-svelte']
```

Comment:
from plugin docs it seems it should work out of the box without adding it to the config, otherwise it didn't run the formatting for me